docs/migrations/js-waku/migration_v0.026_0.027.md

```diff
@@ -0,0 +1,201 @@
+```
+
+### Static Sharding
+
+**Before**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  shardInfo: {
+    clusterId: 1,
+    shards: [0, 1, 2, 3]
+  }
+});
+```
+
+**After**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  networkConfig: {
+    clusterId: 1,
+    shards: [0, 1, 2, 3]
+  }
+});
+```
+
+
+### Auto Sharding
+
+**Before**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  shardInfo: {
+    clusterId: 1,
+    contentTopics: ["/my-app/1/notifications/proto"]
+  }
+});
+```
+
+**After**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  networkConfig: {
+    clusterId: 1,
+    contentTopics: ["/my-app/1/notifications/proto"]
+  }
+});
+```
+
+## Pubsub Topic Configuration
+
+Named pubsub topics are no longer supported. You must use either Static Sharding or Auto Sharding to configure pubsub topics.
+
+**Before**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  pubsubTopics: ["/waku/2/default-waku/proto"]
+});
+```
+
+**After**
+
+Use Static Sharding:
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  networkConfig: {
+    clusterId: 1,
+    shards: [0, 1, 2, 3, 4, 5, 6, 7]
+  }
+});
+```
+
+Or use Auto Sharding:
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  networkConfig: {
+    clusterId: 1,
+    contentTopics: ["/your-app/1/default/proto"]
+  }
+});
+```
+
+## Removed APIs
+
+The following APIs have been removed:
+
+- ApplicationInfo type: Use `string` for application and version in `NetworkConfig` instead.
+- `shardInfo` option in `createLightNode`: Use `networkConfig` instead.
+- `pubsubTopics` option in `createLightNode`: Use `networkConfig` with Static Sharding or Auto Sharding instead.
+
+If you were using `ApplicationInfo` before, you should now use `ContentTopicInfo` (Auto Sharding) and specify your application and version in the content topic `string`.
+
+**Before**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  shardInfo: {
+    clusterId: 1,
+    application: "my-app",
+    version: "1"
+  }
+});
+```
+
+**After**
+```typescript
+import { createLightNode } from "@waku/sdk";
+
+const waku = await createLightNode({
+  networkConfig: {
+    clusterId: 1,
+    contentTopics: ["/my-app/1/default/proto"]
+  }
+});
+```
+
+## Type Changes
+
+- `ShardingParams` has been removed. Use `NetworkConfig` instead.
+- `NetworkConfig` is now defined as `StaticSharding` | `AutoSharding`.
+- `StaticSharding` is equivalent to the previous `ShardInfo`.
+- `AutoSharding` is equivalent to the previous `ContentTopicInfo`.
+
+## Internal/Private Utility Function Changes
```

Comment:
I am not sure we need to document them as they are internal and shouldn't be in the `utils` in the first place because we don't want anyone using them.